generatedTypes/src/components/badge/index.d.ts

```diff
@@ -357,7 +343,7 @@ declare const _default: React.ComponentClass<ViewProps & TouchableOpacityProps &
     /**
      * the badge size (default, small)
      */
-    size?: number | "default" | "small" | "pimpleSmall" | "pimpleBig" | "pimpleHuge" | "large" | undefined;
+    size?: number | undefined;
```

Comment:
```suggestion
    size?: number;
```

Reply:
I just deleted the enum, we want to support only number